README.md

```diff
@@ -8,6 +8,17 @@ Please see [psyclone.pdf](psyclone.pdf) in this directory (or on
 more information. If you would prefer to build the documentation,
 please see the [README](doc/README.md) file in the "doc" directory.
 
+# Try it on Binder #
+
+Some of the examples are available as Jupyter notebooks. These may
+be launched using Binder from the links below.
+
+ * Existing Fortran code: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/575_jupyter_egs?filepath=examples%2Fnemo%2Feg4%2Fcopy_stencil.ipynb)
+
+ * Fortran conforming to PSyKAl separation of concerns: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/575_jupyter_egs?filepath=examples%2Fgocean%2Feg1%2Fopenmp.ipynb)
+ 
+ * DAG generation for the above example: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/stfc/psyclone/575_jupyter_egs?filepath=examples%2Fgocean%2Feg1%2Fdag.ipynb)
```

Comment:
The DAG example gave me filled in results with a failure at the last step. When I reran there was no output produced. Is this because it is running PSyclone master and the fix is on this branch? If so, ignore me :-)

Reply:
That's correct. I've altered the 'dag generation' method in this branch to return the graphviz object.